### PR TITLE
Add bulk edition in stocks

### PR DESCRIPTION
**Pull request by nihco2**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add bulk edition in stocks
| Type?         | new feature
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/FF-7
| How to test?  | Compare with mockups and jira specs

### CODE REVIEW

File: admin-dev/themes/new-theme/js/app/widgets/ps-media.vue [hunk 1]
```diff
@@ -23,7 +23,7 @@
  * International Registered Trademark & Property of PrestaShop SA
  *-->
 <template>
-    <div class="media">
+    <div>
```
Comment (aleeks) [inline admin-dev/themes/new-theme/js/app/widgets/ps-media.vue]: Indent here!

File: admin-dev/themes/new-theme/js/app/widgets/ps-number.vue [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2017 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *-->
+ 
+ <template>
```
Comment (aleeks) [inline admin-dev/themes/new-theme/js/app/widgets/ps-number.vue]: Indent here!

**Review by aleeks — DISMISSED** [review]
Comments, @maximebiloe will be more legitimize to make functional comment

File: admin-dev/themes/new-theme/js/app/pages/stock/components/app.vue [hunk 3]
```diff
@@ -125,6 +125,9 @@
       z-index: 3;
       transition: all 0.2s ease;
       height: 20px;
+      .product-actions & {
```
Comment (maximebiloe) [inline admin-dev/themes/new-theme/js/app/pages/stock/components/app.vue]: Are you sure about the & after the class?

File: admin-dev/themes/new-theme/js/app/pages/stock/components/overview/product-line.vue [hunk 4]
```diff
@@ -67,7 +73,7 @@
       </span>
     </td>
     <td class="qty-spinner">
-      <Spinner :product="product" class="float-xs-right" />
+      <Spinner :product="product" class="pull-xs-right" @updateProductQty="updateProductQty" />
```
Comment (maximebiloe) [inline admin-dev/themes/new-theme/js/app/pages/stock/components/overview/product-line.vue]: `pull...` does not work with bootstrap 4, please upgrade your node modules. Also `-xs-` becomes `sm`

**Review by maximebiloe — DISMISSED** [review]

**Comment by aleeks** [comment]
Thank you @nihco2 @maximebiloe @marionf

**Review by aleeks — APPROVED** [review]

**Comment by xBorderie** [comment]
Nice!

**Pull request merged**